src/stores/GuildChannelStore.js

```diff
@@ -22,6 +22,15 @@ class GuildChannelStore extends DataStore {
     return channel;
   }
 
+  /**
+     * The parent {@link Channel} of the {@link GuildChannel}s in this store *(Only if the store
```

Comment:
Either this looks off on my GH or it is off, but there seems to be an extra space on each line of the docstrings